lib/motion-support/inflector/inflections.rb

```diff
@@ -158,18 +158,31 @@ def irregular(singular, plural)
             singular(/(#{s0})#{srest}$/i, '\1' + srest)
             singular(/(#{p0})#{prest}$/i, '\1' + srest)
           else
-            plural(/#{s0.upcase}(?i)#{srest}$/,   p0.upcase   + prest)
-            plural(/#{s0.downcase}(?i)#{srest}$/, p0.downcase + prest)
-            plural(/#{p0.upcase}(?i)#{prest}$/,   p0.upcase   + prest)
-            plural(/#{p0.downcase}(?i)#{prest}$/, p0.downcase + prest)
-
-            singular(/#{s0.upcase}(?i)#{srest}$/,   s0.upcase   + srest)
-            singular(/#{s0.downcase}(?i)#{srest}$/, s0.downcase + srest)
-            singular(/#{p0.upcase}(?i)#{prest}$/,   s0.upcase   + srest)
-            singular(/#{p0.downcase}(?i)#{prest}$/, s0.downcase + srest)
+            # define_plurals and define_singulars arose from a strange error where
```

Comment:
Sounds like RMA bugs. If your solution solves it I'm happy. I will test in a sec.